include/hal/library/cryptlib/cryptlib_rsa.h

```diff
@@ -35,6 +35,25 @@ typedef enum {
  **/
 extern void *libspdm_rsa_new(void);
 
+/**
+ * Generates RSA context from DER-encoded public key data.
```

Comment:
DER is just encoding.

I think we need clarify which standard we are following for the RSA public key format.

Reply:
The description is added to doc/user_guide.md
> The public key is ASN.1 DER-encoded as [RFC7250](https://www.rfc-editor.org/rfc/rfc7250) describes,
    namely, the `SubjectPublicKeyInfo` structure of a X.509 certificate.

I can add such comment to here as well if you insist.